### PR TITLE
Add contributing mo

### PR DESCRIPTION
**Pull request by yleflour**

### CODE REVIEW

File: contributing/contributing.mo.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## [MO] Contributing
+
+## Owner: [Yann Leflour](https://github.com/yleflour)
+
+## Prerequisites:
+
+- You have cloned [the repository](https://github.com/bamlab/dev-standards) locally
+- You have a markdown/text editor
```
Comment (tychota) [inline contributing/contributing.mo.md]: For example => link to vscode mo

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/contributing-mo/contributing/standard.s.md">standard.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/contributing-mo/contributing/standard.s.md">standard.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/contributing-mo/contributing/contributing.mo.md">contributing.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: contributing/contributing.mo.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+- For a Method of Operation please read [the following standard](/contributing/mo.s.md)
+- For a Standard please read [the following standard](/contributing/standard.s.md)
+- Create a branch and a pull request
+
+> **Check:** Your article is visible in the [pull requests list](https://github.com/bamlab/dev-standards/pulls)
+
+### 2. Review *(~3min)*
+
+- Assign 2 contributors for review
```
Comment (tychota) [inline contributing/contributing.mo.md:22]: Which one, how to choose ? we should use https://github.com/blog/2392-introducing-code-owners

File: contributing/contributing.mo.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Answer the comments which don't need fixing
+- Add the *Review Required* label
+
+> **Check:** Every comment is either deprecated or answered on Github and the label is *Review Required*
+
+### 3. Publishing *(~0min)*
+
+- Once all reviews are ok, the last reviewer will merge and delete your pull request
+
+> **Check:** Your article can be found on [the Gitbook website](https://bamtech.gitbooks.io/dev-standards/)
```
Comment (tychota) [inline contributing/contributing.mo.md]: No troubleshoting ? what if it don't deploy ?

**Review by tychota — APPROVED** [review]
Minor review comments

**Pull request merged**